packages/react-liveness/src/components/FaceLivenessDetector/shared/__tests__/FaceLivenessErrorModal.test.tsx

```diff
@@ -21,7 +21,7 @@ const {
 } = defaultErrorDisplayText;
 
 describe('FaceLivenessErrorModal', () => {
-  it('should render the component content appropriately', () => {
+  it('should render the component content sappropriately', () => {
```

Comment:
Lol was this intentional?

Reply:
Whoops haha, it was intentional to force type checking on a commit. Meant to remove it

Reply:
```suggestion
  it('should render the component content appropriately', () => {
```